library/CMakeLists.txt

```diff
@@ -96,6 +96,13 @@ if(WIN32)
     set(libs ${libs} ws2_32)
 endif(WIN32)
 
+if(APPLE)
```

Comment:
At the moment I have to specify to CMake that this is an Apple build with `cmake -DAPPLE=On .`. That shouldn't be necessary, and I think it should autodetect OSX. Googling the problem, I think you should have platform detection, something like:
```
string(REGEX MATCH "Darwin" CMAKE_PLATFORM_IS_OSX "${CMAKE_SYSTEM_NAME}")
```
And then use `CMAKE_PLATFORM_IS_OSX` in place of `APPLE`.

Our CMake files can't cross compile unless you specify IAR as the compiler, so there should be no similar issue as with the GNU Makefiles.